### PR TITLE
Inidividual Ministry page layout

### PR DESCRIPTION
**Pull request by Alexandra-Green**
## What this PR accomplishes

Aligns the individual Ministry page layout with the Figma, specifically the header and lead. 

## Issue(s) addressed

Removes the RSS box
Adds page header and Lead 
Search bar separates lead and dataset list

## What needs review

- Fonts for the header and lead are the same as Figma
- RSS box is removed
- Navigating to the activity stream and about tabs, the pages look normal
- No missing translations
- "Edit ministry" admin page and individual dataset page Ministry sidebar boxes should look consistent / unchanged / user friendly

**This one should be merged last**

### CODE REVIEW

**Comment by KatiRG** [comment]
"ministère" should be capitalized:
Screen Shot 2023-04-19 at 10 30 53

File: ckanext/ontario_theme/templates/internal/organization/read_base.html [hunk 1]
```diff
@@ -1,28 +1,57 @@
...
-  <li id="ministry-name"
-      ministry-name="{{ c.group_dict.title }}"
-      ministry-name-fr="{{ h.ontario_theme_get_translated_lang(c.group_dict, 'title', 'fr') }}"
-      class="active">
-    {% link_for h.get_translated(c.group_dict,"title")|truncate(35), named_route='organization.read', id=c.group_dict.name %}
-  </li>
+  <li id="ministry-name" ministry-name="{{ c.group_dict.title }}" ministry-name-fr="{{ h.ontario_theme_get_translated_lang(c.group_dict, 'title', 'fr') }}" class="active">{% link_for h.get_translated(c.group_dict,"title")|truncate(35), named_route='organization.read', id=c.group_dict.name %}</li>
 {% endblock breadcrumb_content %}
 
-{% block content_action %}
```
Comment (KatiRG) [inline ckanext/ontario_theme/templates/internal/organization/read_base.html:17]: What was all this code that you removed?

**Review by KatiRG — COMMENTED** [review]
Other than the one comment, it looks good!

File: ckanext/ontario_theme/templates/internal/organization/read_base.html (hunk 1 above)
Comment (Alexandra-Green) [reply ckanext/ontario_theme/templates/internal/organization/read_base.html:17]: It was relocated to the new primary block (top of file), which overrides the primary block in `page.html` & CKAN to allow for the layout change. The empty blocks in {% block primary %} are being filled by `read.html` in this folder.

**Review by KatiRG — APPROVED** [review]

**Pull request merged**